contracts/test/ERC20toCW20PointerTest.js

```diff
@@ -93,12 +93,23 @@ describe("ERC20 to CW20 Pointer", function () {
                     expect(await pointer.balanceOf(sender.evmAddress)).to.equal(balances.account0);
                     expect(await pointer.balanceOf(recipient.evmAddress)).to.equal(balances.account1);
 
+                    const blockNumber = await ethers.provider.getBlockNumber();
                     const tx = await pointer.transfer(recipient.evmAddress, 1);
                     await tx.wait();
 
                     expect(await pointer.balanceOf(sender.evmAddress)).to.equal(balances.account0-1);
                     expect(await pointer.balanceOf(recipient.evmAddress)).to.equal(balances.account1+1);
 
+                    // check logs
+                    const filter = {
+                        fromBlock: blockNumber,
+                        toBlock: 'latest',
+                        address: await pointer.getAddress(),
+                        topics: [ethers.id("Transfer(address,address,uint256)")]
+                    };
+                    const logs = await ethers.provider.getLogs(filter);
+                    expect(logs.length).to.equal(1);
```

Comment:
it may be good just to assert the expected properties of that log (just to make sure the addresses are right, and the amounts are in the right unit)